pages/course/[id].tsx

```diff
@@ -0,0 +1,29 @@
+import type { NextPage } from "next";
+import { useRouter } from "next/router";
+import { useCoursePostsQuery, useCourseQuery } from "@lib/hooks/course";
+import { Text } from "@chakra-ui/react";
+import Layout from "@components/Layout";
+import { BookCardList, PostCardList } from "@components/CardList";
+
+const CoursePage: NextPage = () => {
+  const router = useRouter();
+  const { id } = router.query;
```

Comment:
We should consider changing this to be the course code (i.e. "SFWRENG-2C03") instead in the future, but this is okay for now (since we'd have to make API changes)

Reply:
Created a new Issue for this